app/models.py

```diff
@@ -17,13 +17,9 @@
 class User(AbstractUser):
     """
     Custom user model to use in the future.
-    Email is required to create an User.
     """
-
-    email = models.EmailField(null=False, unique=True, blank=False)
```

Comment:
How would password reset work if you deleted the email field?

Reply:
django allauth do that you dont need email field

Reply:
ok checked and password reset works and email field is available.